pages/services/include/api-reference.tmpl

```diff
@@ -0,0 +1,213 @@
+The DC/OS {{ model.techName }} Service implements a REST API that may be accessed from outside the cluster. The <dcos_url> parameter referenced below indicates the base URL of the DC/OS cluster on which the {{ model.techName }} Service is deployed.
```

Comment:
Does this mean we have to use `model.` in our service templates too?

Reply:
Yes. The merge script handles this change theoretically